stories/SetContactPropertyBlock.stories.ts

```diff
@@ -50,33 +52,36 @@ export const Default = () => (DefaultClass)
 })
 class ExistingDataBlockClass extends BaseMountedVueClass {
   async mounted() {
-    await this.baseMounted(BLOCK_TYPE, SetContactPropertyStore)
+    const {block} = await this.baseMounted(BLOCK_TYPE, SetContactPropertyStore)
+    const blockId = block.uuid
 
-    // Add sample data
-    this.addSubscriberPropertyField({
-      property: {
-        id: '123',
-        name: 'name',
-        displayLabel: 'Name',
-      },
-    })
-    this.addSubscriberPropertyField({
-      property: {
-        id: '124',
-        name: 'date_of_birth',
-        displayLabel: 'Date of birth',
-      },
-    })
-    this.addSubscriberPropertyField({
-      property: {
-        id: '125',
-        name: 'comment',
-        displayLabel: 'Comment',
-      },
-    })
+    this.setDescription(blockId)
+    this.block_updateConfigByPath({blockId, path: 'set_contact_property.property_key', value: 'gender'})
+    this.block_updateConfigByPath({blockId, path: 'set_contact_property.property_value', value: 'M'})
   }
 
   @Mutation addSubscriberPropertyField!: ({property}: { property: any }) => void
```

Comment:
We don't need this anymore, will remove it and the one in below story before merge